src/app/models/error-response.ts

```diff
@@ -0,0 +1,4 @@
+export class ErrorResponse {
```

Comment:
You haven't used this class anywhere.
Add login response class and inherit it from ErrorResponse.

Reply:
Done